docs/workshop/onboarding.md

```diff
@@ -1,3 +1,74 @@
+
+Select login with `operate-first`, and login into the cluster by using your Github credentials.
+
+![Cluster Landing Page](../assets/images/cluster2.png)
+
+2. **Ceph S3 storage -**
+
+Ceph provides a unified storage service with an object, block, and file interface from a single cluster built from commodity hardware components. Operate First hosts a public ceph storage bucket that is accessible by anyone. You can find the credentials and other information for this bucket below:
+
+
+```
+Endpoint:  https://s3-openshift-storage.apps.smaug.na.operate-first.cloud
+
+Bucket Name: opf-datacatalog
+
+Access Key: INDuwCuYntikCsRQioSk
+
+Secret Key: YjLeEDP+Lj30BJTbzgy84b3pzXPpWtlYnRcMYW2/
+
+Total Size: 15G
+```
+
+
+**_Some things to note:_**
+
+
+
+* Do NOT store any confidential information on this bucket as it is open to the public
+* Always back up all your data regularly, as anyone can go in and wipe it
+3. **Jupyterhub -**
+
+JupyterHub is the best way to serve Jupyter notebook for multiple users. It is a multi-user Hub that spawns, manages, and proxies multiple instances of the single-user Jupyter notebook server. Access to operate first JupyterHub is open to the public. All you need is a GitHub account.
+
+* Our JupyterHub instance on the smaug cluster can be found [here](https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/).
+* Login using the operate-first option, and you will be redirected to the JH spawner UI.
+
+![JH Landing Page](../assets/images/JH_landing_page.png)
+
+* You can select any notebook image that you want to try on, for this workshop, we will be working with the `ocp-ci-analysis` image.
+* Select a suitable size for the server, we would recommend using the `medium` size, and start the server.
+
+4. **Trino -**
+
+[Trino](https://trino.operate-first.cloud/) is a distributed SQL query engine designed to query large data sets distributed over one or more heterogeneous data sources. Since our data is stored in Ceph, the Operate First Trino is currently configured to use the `opf-datacatalog-bucket` S3 bucket.
+
+After extracting and analyzing the data in Jupyterhub, we use Trino to store our datasets in the database to further interact with it via Cloudbeaver and Superset.
+
+5. **Cloudbeaver -**
+
+[CloudBeaver](http://cloudbeaver-opf-trino.apps.smaug.na.operate-first.cloud/) is a popular web application that provides a rich UI for working and interacting with SQL databases. The data we exported by using Trino can be accessed in the [CloudBeaver UI](http://cloudbeaver-opf-trino.apps.smaug.na.operate-first.cloud/) and can be queried by SQL queries.
+
+6. **Superset -**
+
+After creating the tables in Trino, we can start creating our dashboards in Superset. Apache Superset is an open source visualization tool that allows users to create interactive dashboards. [Superset](https://superset.operate-first.cloud/) is very simple to use and requires no coding knowledge.
```

Comment:
Shall we say we require very basic SQL knowledge instead of saying no coding knowledge? wdyt?

Reply:
Yeah, sure we can do that.